scripts/generate-clientset.sh

```diff
@@ -63,20 +63,6 @@ if ${SILENT:-false}; then
   LOGGING_FLAGS=""
 fi
 
-tools=()
-for tool in client-gen deepcopy-gen informer-gen lister-gen conversion-gen; do
-  tools+=("k8s.io/code-generator/cmd/${tool}")
-done
-
-# This should match the k8s.io/code-generator version in go.mod.
-tag="v0.22.2"
-echo "Checking out k8s.io/code-generator at tag ${tag}"
```

Comment:
LGTM.

nit: include this log in Makefile.gen

Reply:
the go install command with version will be printed by make.